### PR TITLE
NO-ISSUES: i18n: Add Chinese translations for BPMN Editor

### PR DESCRIPTION
**Pull request by rieonke**
Hello!
I'm working on Chinese localization. I am continuing to translate other components and BPMN editor is the first one.

This is my first time contributing to KIE project and I'm not familiar with the contribution process. If I'm doing something wrong, please let me know. Thanks!

---

> EDIT BY @tiagobento:
> See the related Jira: https://issues.redhat.com/browse/KOGITO-8811

### CODE REVIEW

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/main/resources/org/kie/workbench/common/stunner/core/resources/i18n/StunnerCoreConstants_zh.properties [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+org.kie.workbench.common.stunner.core.reason=原因
+org.kie.workbench.common.stunner.core.delete=删除
+org.kie.workbench.common.stunner.core.edit=编辑
+org.kie.workbench.common.stunner.core.print=打印
+org.kie.workbench.common.stunner.core.reset=重置
+org.kie.workbench.common.stunner.core.increase=增加
+org.kie.workbench.common.stunner.core.decrease=减小
+org.kie.workbench.common.stunner.core.fit=适应
+org.kie.workbench.common.stunner.core.areYouSure=确定吗?
+org.kie.workbench.common.stunner.core.element_uuid=标志符
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/main/resources/org/kie/workbench/common/stunner/core/resources/i18n/StunnerCoreConstants_zh.properties:32]: Maybe  唯一标识符 is better

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/main/resources/org/kie/workbench/common/stunner/core/resources/i18n/StunnerCoreConstants_zh.properties [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+org.kie.workbench.common.stunner.core.rule.fail=校验失败
+org.kie.workbench.common.stunner.core.rule.element=[{0}]\n {1}
+org.kie.workbench.common.stunner.core.rule.property=属性 '{0}' {1}.
+org.kie.workbench.common.stunner.core.client.mediator.zoomArea=选择要跳转的区域
+
+
+#######################
+# Rule Violation Types
+#######################
+org.kie.workbench.common.stunner.core.rule.violations.ContainmentRuleViolation='{0}' 不能包含候(contain)选角色 '{1}'
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/main/resources/org/kie/workbench/common/stunner/core/resources/i18n/StunnerCoreConstants_zh.properties:45]: '{0}' 不能受候选角色'{1}'的包含关系

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/main/resources/org/kie/workbench/common/stunner/core/resources/i18n/StunnerCoreConstants_zh.properties [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+#######################
+# Toolbar
+#######################
+org.kie.workbench.common.stunner.core.client.toolbox.CopySelection=复制选中
+org.kie.workbench.common.stunner.core.client.toolbox.CutSelection=剪切选中
+org.kie.workbench.common.stunner.core.client.toolbox.PasteSelection=粘贴选中
+org.kie.workbench.common.stunner.core.client.toolbox.ClearShapes=清除形状状态
+org.kie.workbench.common.stunner.core.client.toolbox.ClearDiagram=清除图表
+org.kie.workbench.common.stunner.core.client.toolbox.DeleteSelection=删除选中 [DEL]
+org.kie.workbench.common.stunner.core.client.toolbox.ExportJPG=下载为 JPG
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/main/resources/org/kie/workbench/common/stunner/core/resources/i18n/StunnerCoreConstants_zh.properties:71]: 导出？

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties [hunk 4]
```diff
@@ -0,0 +1,728 @@
...
+# DefinitionSet
+#######################
+org.kie.workbench.common.stunner.bpmn.BPMNDefinitionSet.description=BPMN2
+org.kie.workbench.common.stunner.bpmn.BPMNDefinitionSet.svgNodeId=bpmn2nodeid
+
+#######################
+# Categories
+#######################
+org.kie.workbench.common.stunner.bpmn.definition.BPMNCategories.Containers=容器
+org.kie.workbench.common.stunner.bpmn.definition.BPMNCategories.SubProcess=子过程
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties:12]: 子流程？

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties [hunk 5]
```diff
@@ -0,0 +1,728 @@
...
+org.kie.workbench.common.stunner.bpmn.definition.BPMNCategories.Activities=活动
+org.kie.workbench.common.stunner.bpmn.definition.BPMNCategories.StartEvents=开始事件
+org.kie.workbench.common.stunner.bpmn.definition.BPMNCategories.IntermediateEvents=中间事件
+org.kie.workbench.common.stunner.bpmn.definition.BPMNCategories.EndEvents=结束事件
+org.kie.workbench.common.stunner.bpmn.definition.BPMNCategories.ConnectingObjects=连接对象
+org.kie.workbench.common.stunner.bpmn.definition.BPMNCategories.Gateways=网关
+org.kie.workbench.common.stunner.bpmn.definition.BPMNCategories.CustomTasks=自定义事件
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties:19]: 自定义任务

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties [hunk 6]
```diff
@@ -0,0 +1,728 @@
...
+
+#######################
+# Morph base types
+#######################
+org.kie.workbench.common.stunner.bpmn.definition.BaseTask=任务
+org.kie.workbench.common.stunner.bpmn.definition.BaseStartEvent=开始事件
+org.kie.workbench.common.stunner.bpmn.definition.BaseEndEvent=结束事件
+org.kie.workbench.common.stunner.bpmn.definition.BaseCatchingIntermediateEvent=捕获中间事件
+org.kie.workbench.common.stunner.bpmn.definition.BaseThrowingIntermediateEvent=抛出中间事件
+org.kie.workbench.common.stunner.bpmn.definition.BaseSubprocess=子过程
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties:31]: 流程

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties [hunk 7]
```diff
@@ -0,0 +1,728 @@
...
+org.kie.workbench.common.stunner.bpmn.definition.StartEscalationEvent.description=对升级到组织中的另一个角色做出反应。 此事件仅在事件子流程内部使用。
+org.kie.workbench.common.stunner.bpmn.definition.StartCompensationEvent.title=补偿开始事件
+org.kie.workbench.common.stunner.bpmn.definition.StartCompensationEvent.description=补偿处理。此事件仅在事件子流程内部使用。
+org.kie.workbench.common.stunner.bpmn.definition.ReusableSubprocess.title=可复用子流程
+org.kie.workbench.common.stunner.bpmn.definition.ReusableSubprocess.description=可复用子流程，可以被其它流程调用。
+org.kie.workbench.common.stunner.bpmn.definition.AdHocSubprocess.title=特设子流程
+org.kie.workbench.common.stunner.bpmn.definition.AdHocSubprocess.description=特设子流程
+org.kie.workbench.common.stunner.bpmn.definition.EmbeddedSubprocess.title=嵌入子流程
+org.kie.workbench.common.stunner.bpmn.definition.EmbeddedSubprocess.description=嵌入子流程
+org.kie.workbench.common.stunner.bpmn.definition.EventSubprocess.title=事件子过程
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties:100]: 流程

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties [hunk 8]
```diff
@@ -0,0 +1,728 @@
...
+# Generic Service Task
+#######################
+org.kie.workbench.common.stunner.bpmn.definition.GenericServiceTask.title=服务任务
+org.kie.workbench.common.stunner.bpmn.definition.GenericServiceTask.description=服务任务
+org.kie.workbench.common.stunner.bpmn.definition.property.service.GenericServiceTaskExecutionSet.label=实现/执行
+#######################
+# DiagramSet
+#######################
+org.kie.workbench.common.stunner.bpmn.definition.property.diagram.DiagramSet.label=流程
+org.kie.workbench.common.stunner.bpmn.definition.property.diagram.AdHoc.label=Ad-Hoc
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties:211]: before it translates to 特设流程， keep them consistent. BTW, I think keeping 'Ad-Hoc' is better.

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties [hunk 9]
```diff
@@ -0,0 +1,728 @@
...
+#######################
+# BusinessRuleTaskExecutionSet, EmptyTaskExecutionSet, ReusableSubprocessTaskExecutionSet, UserTaskExecutionSet & AdHocSuprocessExecutionSet.
+#######################
+org.kie.workbench.common.stunner.bpmn.workitem.CustomTaskExecutionSet.label=实现/执行
+org.kie.workbench.common.stunner.bpmn.definition.property.task.BusinessRuleTaskExecutionSet.label=实现/执行
+org.kie.workbench.common.stunner.bpmn.definition.property.task.EmptyTaskExecutionSet.label=Empty 实现/执行
+org.kie.workbench.common.stunner.bpmn.definition.property.task.ReusableSubprocessTaskExecutionSet.label=实现/执行
+org.kie.workbench.common.stunner.bpmn.definition.property.task.ScriptTaskExecutionSet.label=实现/执行
+org.kie.workbench.common.stunner.bpmn.definition.property.task.UserTaskExecutionSet.label=实现/执行
+org.kie.workbench.common.stunner.bpmn.definition.property.task.AdHocSubprocessTaskExecutionSet.label=实现/执行
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties:398]: Ad-Hoc needs to keep them consistent

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties [hunk 10]
```diff
@@ -0,0 +1,728 @@
...
+org.kie.workbench.common.stunner.bpmn.definition.property.collaboration.events.DataExpressionType.label=数据表达式类型
+
+#######################
+# Common Properties
+#######################
+org.kie.workbench.common.stunner.bpmn.definition.property.common.ConditionExpression.label=条件表达式
+org.kie.workbench.common.stunner.bpmn.definition.property.common.ConditionExpression.description=条件表达式
+
+#######################
+# Case Management
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants_zh.properties:528]: https://en.wikipedia.org/wiki/Case_management_(mental_health)
I think keeping it on the English version is better.

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/main/resources/org/kie/workbench/common/stunner/client/widgets/resources/i18n/StunnerWidgetsConstants_zh.properties [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+NameEditBoxWidgetViewImpl.close=关闭
+NameEditBoxWidgetViewImpl.name=名称
+SessionPresenterView.Error=错误
+SessionPresenterView.Warning=警告
+SessionPresenterView.Info=信息
+SessionPresenterView.Notifications=警告面板详情
+
+MarshallingResponsePopup.OkAction=确认
+MarshallingResponsePopup.CancelAction=取消
+MarshallingResponsePopup.CopyToClipboardActionTitle=拷贝消息到粘贴板
```
Comment (bxf12315) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/main/resources/org/kie/workbench/common/stunner/client/widgets/resources/i18n/StunnerWidgetsConstants_zh.properties:20]: 复制消息到剪切板

**Comment by tiagobento** [comment]
Thanks @bxf12315 for kindly making yourself available to review this! :)

**Comment by ederign** [comment]
@bxf12315 when you have a chance, could you please review this ?

**Comment by bxf12315** [comment]
> @bxf12315 when you have a chance, could you please review this ?

Sorry, I forget to submit all comments.

**Comment by tiagobento** [comment]
Closing due to inactivity.

**Pull request closed without merging**